src/man/sssd.conf.5.xml

```diff
@@ -3406,6 +3406,46 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>dns_resolver_server_timeout (integer)</term>
+                    <listitem>
+                        <para>
+                            Defines the amount of time (in milliseconds)
+                            SSSD would try to talk to DNS servers before
+                            trying next DNS server.
+                        </para>
+                        <para>
+                            Please see the section <quote>FAILOVER</quote>
+                            for more information about the service
+                            resolution.
+                        </para>
+                        <para>
+                            Default: 1000
+                        </para>
+                    </listitem>
+                </varlistentry>
+
+                <varlistentry>
+                    <term>dns_resolver_op_timeout (integer)</term>
+                    <listitem>
+                        <para>
+                            Defines the amount of time (in milliseconds)
+                            Defines the amount of time (in seconds) to
+                            wait to try to resolve single DNS query
+                            (e.g. resolution of a hostname or an SRV record)
+                            before try next hostname or DNS discovery.
```

Comment:
There's something strange in that phrase but I don't know what it is. @justin-stephenson what do you think?